ddagent

```diff
@@ -0,0 +1 @@
+./bin/agent/agent $@ -c bin/agent/dist/datadog.yaml
```

Comment:
i'd exclude this file from the PR since it's just a local testing file. 
it can be helpful to add it to your global git ignore.